src/js/utilities/audio.js

```diff
@@ -9,8 +11,16 @@ let createPlayer = () => new Audio();
 
 let playSound = (sound) => {
   let player = createPlayer();
-  player.src = sounds[sound];
+  let soundEffect = sounds[sound];
+  player.src = soundEffect;
   player.play();
+  player.addEventListener(
+    'ended',
+    function(e) {
+      url.revokeObjectURL(soundEffect);
```

Comment:
A bit confused as to what this line is doing. There's no file to release since we're generating the sounds on the fly, right?